src/AppBundle/DataFixtures/ORM/ClubFixtures.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace AppBundle\DataFixtures\ORM;
+
+use AppBundle\DataFixtures\BaseFixture;
+use AppBundle\Factory\ClubFactory;
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use AppBundle\Entity\User;
+
+class ClubFixtures extends BaseFixture implements DependentFixtureInterface
+{
+    public function load(ObjectManager $manager)
+    {
+        foreach (range(1, 2) as $i) {
+            $club = ClubFactory::createClub(
+                $this->faker->company,
+                $this->faker->city,
+                $this->faker->streetAddress,
+                $this->faker->url
+            );
+            if ($i === 1) {
```

Comment:
to complex, I would just add club to user ($user->setClub) not user to club. I think we can remove method from club to add user.

Reply:
Yes, i think about it. I want to add Club fixtures and not to touch User entity. We can have method to add Club to User and also add User to Club. What was earlier ? :) Club or User. It's not relevant.

Reply:
Yes we can have both, but what for? YAGNI Just use $user->setClub

Reply:
Ok, it's make sense. Sometimes I do code from my side.